Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -760,7 +760,7 @@ - (void)addGroupIdToProfileWhitelist:(NSData *)groupId
             NSString *groupIdKey = [groupId hexadecimalString];
 
             // We just consult the lazy cache, not the db.
-            if (self.groupProfileWhitelistCache[groupIdKey]) {
+            if ([self isGroupIdInProfileWhitelist:groupId]) {
```

Comment:
Shoot. I thought I checked for all the similar uses. 

Can you remove the now inaccurate comment?